controller/argument_value_resolver.rst

```diff
@@ -239,6 +239,7 @@ priority of 100. As this one is responsible for fetching attributes from the
 lower priority. This makes sure the argument resolvers are not triggered when
 the attribute is present. For instance, when passing the user along a
 sub-requests.
+However when the injected parameter is an object unrelated to the request (User, Session) it is necessary to set a priority of at least 100 otherwise the resolver will not be thrown.
```

Comment:
That's the opposite actually:
- your resolver does not relate with any attribute of the request (e.g: injecting the current User or session) => below 100 so `RequestAttributeValueResolver` has precedence
- your resolver concerns a request attribute (it's part of the route params in the path or defaults, e.g: transforming uuid string in `foo/{uuid}/bar` to an object) => 100+ to have precedence over `RequestAttributeValueResolver`

Reply:
Okay! I misunderstood